sunpy/coordinates/transformations.py

```diff
@@ -56,6 +56,16 @@ def _carrington_offset(obstime):
     from .ephemeris import get_sun_L0
     return get_sun_L0(obstime)
 
+
+def _observers_are_equal(obs_1, obs_2):
+    if not (isinstance(obs_1, BaseCoordinateFrame) and isinstance(obs_2, BaseCoordinateFrame)):
+        raise ValueError("To compare two observers, both must be instances of BaseCoordinateFrame. "
+                         "Cannot compare two observers {} and {}.".format(obs_1, obs_2))
+    return (quantity_allclose(obs_1.lat, obs_2.lat) and
+            quantity_allclose(obs_1.lon, obs_2.lon) and
+            quantity_allclose(obs_1.radius, obs_2.radius))
```

Comment:
If observers have same lat, lon and radius, then they are equal? Is that for every type of frame?